src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesArray = sourceString.split(';');
+  const styleObject = {};
```

Comment:
[CODE STYLE] - Avoid variable namings like `styleObject`, as it will decrease code readability. Consider naming variables to reflect their content or purpose rather than their data type, for example `styleMap` or `cssStyles`.